bin/brew

```diff
@@ -23,7 +23,7 @@ unset GEM_HOME
 unset GEM_PATH
 
 BREW_SYSTEM=$(uname -s | tr "[:upper:]" "[:lower:]")
-if [ "$BREW_SYSTEM" = "darwin" ]
+if [ "$BREW_SYSTEM" = "darwin" ] && [ -z "$HOMEBREW_OLD_RUBY_TEST" ]
```

Comment:
That's cool, something we can do later. Unrelated but I think while we move to Ruby 2 we should also consider deprecating <=10.8 support (and/or moving to Tigerbrew)